app/components/DepositWithdraw/blocktrades/OpenledgerGatewayDepositRequest.jsx

```diff
@@ -10,7 +10,7 @@ import AccountBalance from "../../Account/AccountBalance";
 import AssetName from "components/Utility/AssetName";
 import LinkToAccountById from "components/Utility/LinkToAccountById";
 import {requestDepositAddress, getDepositAddress} from "common/gatewayMethods";
-import {blockTradesAPIs, openledgerAPIs} from "api/apiConfig";
+import {blockTradesAPIs, openledgerAPIs, bitsparkAPIs} from "api/apiConfig";
```

Comment:
Why was OpenledgerGatewayDepositRequest adjusted for BitSpark?

Reply:
We've started work while ago based on `BlockTradesGateway.jsx` implementation when everything was coupled together. Right now it seems logic were moved to  Openledger files after rebase. I've already asked folks update codebase to reflect latest changes & have all changes in separate file.

Reply:
Thanks, I will wait for the update.

If you finish it until tomorrow (11am UTC+2), then I can include it in next staging release, otherwise it will have to wait until next one.

Reply:
@sschiessl-bcp we've updated the merge request with updated changes. 
- this branch is based on fresh develop (no need to merge it again)
- branch was deployed to https://dex-preview.bitspark.io/, so you can easily check address generation
- CI is failing at the same stage as in `develop` branch (https://ci.appveyor.com/project/svk31/bitshares-ui/build/1.0.2603)
- I've run locally `npm install --production && npm ci`, it solved existing issue with `npm ci ` in develop

Reply:
Yes, I saw that yesterday as well, it was the react-copy-to-clipboard dependency